lib/ansible/galaxy/collection/__init__.py

```diff
@@ -333,11 +333,19 @@ def verify_local_collection(local_collection, remote_collection, artifacts_manag
                 os.path.join(b_collection_path, to_bytes(name, errors='surrogate_or_strict'))
             )
 
+    b_ignore_patterns = [
+        b'*.pyc',
+    ]
+    b_ignore_dirs = frozenset([b'__pycache__',])
```

Comment:
Since `*.pyc` files are being ignored, isn't it unnecessary to also ignore `__pycache__` directories?